test/controllers/DeparturesControllerSpec.scala

```diff
@@ -247,6 +251,7 @@ class DeparturesControllerSpec extends SpecBase with ScalaCheckPropertyChecks wi
 
           val result = route(application, request).value
 
+          contentAsString(result) mustEqual "Failed to create Departure with the following error: LocalDateParseFailure(Failed to parse DatOfPreMES9 to LocalDate with error: Text '' could not be parsed at index 0)"
```

Comment:
This error message generated could be changed to the following:<br>`The value of element 'DatOfPreMES9' is not valid with respect to pattern 'yyyyMMdd'`